cmd/agent/subcommands/launchgui/command.go

```diff
@@ -50,14 +50,14 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 	return []*cobra.Command{launchCmd}
 }
 
-func launchGui(_ config.Component, _ *cliParams) error {
+func launchGui(config config.Component, _ *cliParams) error {
 	guiPort := pkgconfig.Datadog.GetString("GUI_port")
 	if guiPort == "-1" {
 		return fmt.Errorf("GUI not enabled: to enable, please set an appropriate port in your datadog.yaml file")
 	}
 
 	// Read the authentication token: can only be done if user can read from datadog.yaml
-	authToken, err := security.FetchAuthToken()
+	authToken, err := security.FetchAuthToken(pkgconfig.Datadog)
```

Comment:
```suggestion
	authToken, err := security.FetchAuthToken(config)
```

Best to avoid adding references to the global config object when possible.

Reply:
Definitely, I tried using the config object when I noticed one but I guess I missed some ! Nice catch !